src/shared/services/bookmarks-views-plays-service/bookmarks-views-plays-service.const.ts

```diff
@@ -123,6 +155,14 @@ export const GET_EVENT_QUERIES: () => {
 			getResponseCount: (response: GetCollectionViewCountQuery): number =>
 				response.app_collections[0]?.view_counts?.[0]?.count || 0,
 		},
+		// Added empty event for future use and solving typing issues
```

Comment:
reference the jira ticket and make this a // TODO
or if no jira ticket exists, make one under the "delen en samenwerken" epic, and then reference it